cli/src/supervisor.rs

```diff
@@ -49,8 +50,10 @@ impl Do<DashboardEvent> for Supervisor {
     async fn handle(&mut self, event: DashboardEvent, ctx: &mut ActorContext<Self>) -> Result<(), Self::Error> {
         match event {
             DashboardEvent::Terminated => {
+                log::debug!("[Shutdown] `DashboardEvent::Terminatedd` received");
                 self.dashboard.take();
                 ctx.shutdown();
+                tokio::time::sleep(Duration::from_secs(10)).await;
```

Comment:
Is there an event we can wait for instead of this fixed time?

Reply:
For now, this seems to work fine. I did start a separate branch to get a tokio oneshot channel going as I could not find any events wired into here, but that will take some time to complete.

Reply:
Agree with Mike. You can get feedback that the docker process has exited, rather than wait for nnn seconds to close the app

Reply:
tbc, I'm ok to merge this behaviour now, but really need to get the proper shutdown behaviour in as a follow up

Reply:
I expanded the Docker status mapping and now properly handle `ContainerState::NotFound` during shutdown, which in turn enabled me to remove the _sleep-when-exit statements_ in the launchpad front end.